xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -541,7 +570,7 @@ final class DatabaseService {
         
         let results = try database.execute(
             sql: """
-            SELECT id, modified, excerpt
+            SELECT did, id, modified, excerpt
```

Comment:
I would like to add a test for `listRecentMemos` that ensures we're getting the data out of the DB correctly.

Reply:
This is covered by `Tests_DatabaseService.testListRecentMemos` & `Tests_DatabaseService.testListRecentMemosWithoutOwner` but I've added a few more assertions.